test/functional/basic/find-e2e-specs.js

```diff
@@ -400,4 +404,53 @@ describe('XCUITestDriver - find', function () {
       await els[0].getAttribute("type").should.eventually.eql("XCUIElementTypeTable");
     });
   });
+
+  describe('by image', function () {
+    before(async function () {
+      // use the driver settings that cause the most code paths to be exercised
+      await driver.updateSettings({
+        fixImageTemplateSize: true,
+        autoUpdateImageElementPosition: true,
+      });
+    });
+    it('should find image elements', async function () {
+      let els = await driver.elementsByImageFile(UICATALOG_LIST_IMG);
+      els.should.have.length(1);
+    });
+    it('should find an image element', async function () {
+      let el = await driver.elementByImageFile(UICATALOG_LIST_IMG);
+      el.value.should.match(/appium-image-element/);
+    });
+    it('should not find an image element that is not matched', async function () {
+      await driver.elementByImageFile(TESTAPP_BUTTON_IMG)
+        .should.eventually.be.rejectedWith(/Error response status: 7/);
+    });
+    it('should find anything with a threshold low enough', async function () {
+      const {imageMatchThreshold} = await driver.settings();
+      await driver.updateSettings({imageMatchThreshold: 0});
+      await driver.elementByImageFile(TESTAPP_BUTTON_IMG).should.eventually.exist;
+      await driver.updateSettings({imageMatchThreshold});
+    });
+    it('should be able to get basic element properties', async function () {
+      let el = await driver.elementByImageFile(UICATALOG_LIST_IMG);
+      await el.isDisplayed().should.eventually.be.true;
+      let size = await el.getSize();
+      size.width.should.be.above(0);
+      size.height.should.be.above(0);
+      let loc = await el.getLocation();
+      (loc.x >= 0).should.be.true;
```

Comment:
Does `loc.x.should.be.above(0);` not work? Seems strange to have two entirely different ways of doing the same thing in one test.

Reply:
well I didn't want it to be above 0, i wanted it to be greater than or equal to zero, and i was too lazy to look up the chai docs to see how to make that work.

Reply:
but I just did and it is `be.at.least`, so i will change it